### PR TITLE
Search

### PR DESCRIPTION
**Pull request by varungu**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @varungu has signed the CLA. Thanks for the pull request!

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/BoxSearchView.java [hunk 1]
```diff
@@ -45,6 +46,10 @@ private void initSearchView(final Context context){
         final ImageView searchCloseButton = (ImageView) searchPlate.findViewById(R.id.search_close_btn);
         searchCloseButton.setImageResource(R.drawable.ic_clear_gray_24dp);
 
+
+        EditText editText = ((EditText)findViewById(R.id.search_src_text));
```
Comment (doncung) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/uidata/BoxSearchView.java]: Add version checking here for kitkat below only.

**Comment by doncung** [comment]
+1

File: box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxSearchFragment.java [hunk 2]
```diff
@@ -121,7 +121,7 @@ public void onClick(View v) {
             }
         });
         setupSearchFiltersHeader();
-
+        search();
```
Comment (doncung) [inline box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxSearchFragment.java]: Move to onResume.

**Pull request merged**